### DISCUSSION TITLE
notify alert

### DISCUSSION THREAD

**Discussion by alexeysvyridov** [WeblateOrg/weblate#5568] [Q&A] [answered]
Hi, 3 days ago I had a notify **"alert"**,  and now i can see only **9%** translations in the components.

what is it ? why it is happened? and how to fix that? please help  
Broken project website URL
This project is set up incorrectly.

The project website URL points to a non-existing location:

https://dev1.vitalerter.com (HTTPSConnectionPool(host='dev1.vitalerter.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f57ac192e80>: Failed to establish a new connection: [Errno 111] Connection refused')))

**Comment by nijel** [comment, marked as answer]
The alert has nothing to do with status of the translation, it just tells that Weblate server could not access the website. See https://docs.weblate.org/en/latest/devel/alerts.html for more info on alerts.

**Reply by alexeysvyridov** [reply]
understood, can you explained, what might be the reason, this error?

**Reply by nijel** [reply]
The error says it: "Connection refused". It can be a networking issue or the server was down when it was checked.
